src/lib/jsonParse/jsonUpdateContextAttributeRequest.cpp

```diff
@@ -63,6 +63,9 @@ static std::string attributeValue(const std::string& path, const std::string& va
   //       save the 'typeFromXmlAttribute', a ContextAttribute has been added to 
   //       UpdateContextAttributeData.
   //
+  // FIXME P4 #1862: typeFromXmlAttribute was removed. The comment abov is obsolete and maybe
```

Comment:
Perhaps/Probably the entire 'lastContextAttribute mechanism' can be removed.
We need to see exactly where this pointer is used.